lib/src/authentication/src/auth.dart

```diff
@@ -1,5 +1,4 @@
-import 'common.dart';
-import 'rest/options.dart';
+import '../authentication.dart';
```

Comment:
I think the repetition of `src` is just the outcome of having modules inside modules whilst trying to avoid moving **every** file. Perhaps we can remove the first `src` folder from the file paths. WDYT?

This is what I meant by 
> This is a pattern observed in the [getx repo](https://github.com/jonataslaw/getx/tree/master/lib), though if another pattern is cleaner, please mention it.

Reply:
> Perhaps we can remove the first src folder from the file paths. WDYT?
I suggest we move the files after the PN PR is merged.

Reply:
> Perhaps we can remove the first src folder from the file paths

Sounds perfect.

> I suggest we move the files after the PN PR is merged.

Sure